web/models/epjs_user.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Healthlocker.EPJSUser do
+  use Healthlocker.Web, :model
+
+  schema "epjs_users" do
+    field :patient_id, :integer
```

Comment:
Just thinking that in the ePJS DB this is actually `Patient_ID`, so we might have to use a string here and keep the casing consistent?